css/styles.css

```diff
@@ -0,0 +1,480 @@
+}
+
+/* Main-Header Section Ends */
+
+/* Form Main Begins */
+.sub-header {
+  padding: 0;
+  margin: auto;
+}
+
+.mini-main-header {
+  padding-top: 0;
+  text-align: center;
+}
+
+.mini-logo-container {
+  margin: 0 auto;
+  display: block;
+  width: 100%;
+}
+
+.logo-list {
+  margin-left: auto;
+  margin-right: auto;
+  list-style: none;
+  margin-bottom: 15px;
+}
+
+.logo-list li {
+  margin-right: 9px;
+  cursor: pointer;
+}
+
+/* mint-icon */
+.form-main .mint-icon {
+  width: 23px;
+  height: 23px;
+  display: inline-block;
+  margin-right: 4px;
+  position: relative;
+  cursor: pointer;
+  background:
+    url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyMCAyMCI+PGcgZmlsbD0ibm9uZSI+PGNpcmNsZSBjeD0iMTAiIGN5PSIxMCIgcj0iMTAiIGZpbGw9IiMwMEE2QTQiLz48cGF0aCBmaWxsPSIjRkZGIiBkPSJNMTAgMi43NzdjLS42NTguNjYtNS4xNzQgNS4zMDktNS4xNzQgOC45MyAwIDIuNTc4IDIuMDU1IDMuNDMyIDUuMTc0IDUuNTU0IDMuMTItMi4xMjIgNS4xNzQtMi45NzYgNS4xNzQtNS41NTQgMC0zLjYyMS00LjUxNS04LjI3LTUuMTc0LTguOTN6bTAgMTIuNzQ4bC0uNjA3LS4zOTdjLTIuMzItMS41MTEtMy4xMjMtMi4xMDEtMy4xMjMtMy40MjEgMC0yLjA0NCAyLjAyOC00Ljg4OCAzLjczLTYuODE2djEwLjYzNHoiLz48L2c+PC9zdmc+)
+    no-repeat;
+}
+
+.form-main .mint-text {
+  display: inline-block;
+  position: relative;
+  background:
+    url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzNyIgaGVpZ2h0PSIxNCI+PHBhdGggZmlsbD0iIzZCNkM3MiIgZD0iTTE0LjMzMiA3LjkyN3Y1LjUzN0gxMy4wMlY4LjQ2NmMwLTEuODktLjYzNi0zLjE2NC0yLjMzNS0zLjE2NC0xLjg5IDAtMi41NjYgMS42NzktMi41NjYgMy4zNTh2NC44MDRINi44MDhWOC4xOTZjMC0xLjc1Ni0uNDI1LTIuODk0LTIuMTgxLTIuODk0LTEuMjU0IDAtMi42MjQgMS4xMi0yLjYyNCAzLjUzMXY0LjYzMUguNjlWNi40MjJjMC0uNDgzLS4wMi0xLjM3LS4wNzgtMS45ODhoMS4yNTVjLjAzOC40ODIuMDc3IDEuMTU3LjA3NyAxLjU0NGguMDM4Yy41MjEtMS4wNDMgMS42OC0xLjc3NiAyLjk3Mi0xLjc3Ni45ODQgMCAyLjE4LjM2NyAyLjc3OCAxLjkxLjYxNy0xLjIzNCAxLjcxNy0xLjkxIDMuMTY0LTEuOTEgMi4yOTcgMCAzLjQzNSAxLjQ4NiAzLjQzNSAzLjcyNXptMi44MzkgNS41MzdoMS4zMTJ2LTkuMDNIMTcuMTd2OS4wM3ptLjY1Ny0xMy40M2MtLjQ2NCAwLS45MjcuMzg3LS45MjcuOTI3cy40NjMuOTI1LjkyNy45MjVjLjQ2MyAwIC45MjUtLjM4NS45MjUtLjkyNWEuOTM4LjkzOCAwIDAgMC0uOTI1LS45Mjd6bTguMDQ0IDQuMTY4Yy0xLjI3NCAwLTIuNTg2LjczMy0zLjA4NyAxLjc3NmgtLjA0YzAtLjM4Ny0uMDM3LTEuMDYyLS4wNzctMS41NDRoLTEuMjUzYy4wNTguNjE4LjA3NyAxLjUwNS4wNzcgMS45ODh2Ny4wNDJoMS4zMTJ2LTQuNjNjMC0yLjQxMiAxLjM1LTMuNTMyIDIuNzQtMy41MzIgMS44OTEgMCAyLjQ1IDEuMjczIDIuNDUgMy4xNjR2NC45OThoMS4zMTNWNy45MjdjMC0yLjI0LTEuMTQtMy43MjUtMy40MzUtMy43MjV6bTEwLjgxNCAxLjMzMnYtMS4xSDM0LjE0VjEuODg2aC0xLjMxMnYyLjU0OGgtMS44N3YxLjFoMS44N3Y1LjYzNGMwIDEuOTg3IDEuMzUyIDIuNDEyIDIuMzE2IDIuNDEyLjU5OSAwIDEuMDgxLS4wOTYgMS41NDMtLjI3bC0uMDU2LTEuMTM5Yy0uMzUuMTc0LS43NzMuMjktMS4xNC4yOS0uNzkxIDAtMS4zNS0uMzA5LTEuMzUtMS42NlY1LjUzNGgyLjU0NnoiLz48L3N2Zz4=)
+    no-repeat;
+  width: 37px;
+  height: 20px;
+  cursor: pointer;
+}
```

Comment:
- [ ] **MANDATORY** For good coding practices, don't repeat yourself. You are using some repeated attributes for your product's images and texts. For solving that you can use multiple classes for any element, so for example,

  If you use these lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
   and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px, color of #fff and background-color of #000 -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px, color of #fff and background-color of #000 -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px, color of #fff and background-color of #000 -->
  ...
  ```

Reply:
Done.